src/java.net.http/share/classes/module-info.java

```diff
@@ -25,7 +25,127 @@
+ * jdk.httpclient.keepalive.timeout.h2 property.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.keepalive.timeout.h2}</b> (default: see below)<br>
+ * The number of seconds to keep idle HTTP/2 connections alive. If not set, then the
+ * jdk.httpclient.keepalive.timeout setting is used.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.maxframesize}</b> (default: 16384 or 16kB)<br>
+ * The HTTP/2 client maximum frame size in bytes. The server is not permitted to send a frame larger than this.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.maxstreams}</b> (default: 100)<br>
+ * The maximum number of HTTP/2 streams per connection.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.receiveBufferSize}</b> (default: operating system default)<br>
+ * The HTTP client socket receive buffer size in bytes.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.redirects.retrylimit}</b> (default: 5)<br>
+ * The maximum number of attempts to send a HTTP request when redirected or any failure occurs for any reason.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.websocket.writeBufferSize}</b> (default: 16384 or 16kB)<br>
+ * The buffer size used by the web socket implementation for socket writes.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.windowsize}</b> (default: 16777216 or 16 MB)<br>
+ * The HTTP/2 client stream window size in bytes.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.auth.retrylimit}</b> (default: 3)<br>
+ * The number of attempts the Basic authentication filter will attempt to retry a failed authentication.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.internal.httpclient.selectorTimeout}</b> (default: 3000)<br>
+ * The selector thread wakes up repeatedly after this number of milliseconds when there is no activity.
+ * This allows the thread to terminate when no longer needed. The value provided is limited
+ * to the range from 1 to 1200(???) seconds.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpclient.sendBufferSize}</b> (default: operating system default)<br>
+ * The HTTP client socket send buffer size. Values less than or equal to zero are ignored.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.internal.httpclient.disableHostnameVerification}</b> (default: false)<br>
+ * If true, hostname verification in SSL certificates is disabled. This is a system property only.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.http.auth.proxying.disabledSchemes}</b> (default: see conf/net.properties)<br>
+ * A comma separated list of HTTP authentication scheme names, that are disallowed for use by
+ * the HTTP client implementation, for HTTP proxying.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.http.auth.tunneling.disabledSchemes}</b> (default: see conf/net.properties)<br>
+ * A comma separated list of HTTP authentication scheme names, that are disallowed for use by
+ * the HTTP client implementation, for HTTP CONNECT tunneling.
+ * </li>
```

Comment:
Should we provide a link to net-properties.html here?

Reply:
I don't see the benefit in that?

Reply:
I believe these properties are documented there already.

Reply:
Ah - no. They are not. My mistake.